package.json

```diff
@@ -191,7 +191,7 @@
 		"build:package-types": "node ./bin/packages/validate-typescript-version.js && tsc --build",
 		"build": "npm run build:packages && wp-scripts build",
 		"check-engines": "wp-scripts check-engines",
-		"check-licenses": "concurrently \"wp-scripts check-licenses --prod --gpl2\" \"wp-scripts check-licenses --dev\"",
+		"check-licenses": "wp-scripts check-licenses --dev",
```

Comment:
It seems like it shouldn't be included here. There is this fundamental difference in approach to licensing for production code and development tools.

Reply:
I did this change to debug the failing dev command on Travis CI. Reverted now 👍